doc/classes/CanvasItem.xml

```diff
@@ -525,6 +525,7 @@
 			<return type="bool" />
 			<description>
 				Returns [code]true[/code] if the node is present in the [SceneTree], its [member visible] property is [code]true[/code] and all its ancestors are also visible. If any ancestor is hidden, this node will not be visible in the scene tree, and is therefore not drawn (see [method _draw]).
+				Parent visibility checks include only nodes that inherit from [CanvasItem], [CanvasLayer] and [Window]. If the parent is of another type (e.g. [Node], [AnimationPlayer] or [Node3D]), parent visibility is assumed to be [code]true[/code].
```

Comment:
Nitpicks since https://github.com/godotengine/godot/pull/93735 exists, to make this a bit more clear hopefully.
"Parent visibility checks" and "parent visibility" makes it seem more complicated than it actually is
```suggestion
				This method includes only nodes that inherit from [CanvasItem], [CanvasLayer], and [Window]. If the parent is of any other type (such as [Node], [AnimationPlayer], or [Node3D]), the parent is assumed to be visible.
```
Alternatively the wording could make the behavior clearer ("propagating up" is a thing?)
```suggestion
				The check propagates only through parent nodes that inherit from [CanvasItem], [CanvasLayer], and [Window]. A parent of any other type stops the propagation and is assumed to be visible.
```

Reply:
I altered it a bit, is it good enough?